dev/run_benchmarks.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Check if the file exists before removing it
+if [ -f "tmp/codeowners_benchmarks.md" ]; then
+  rm tmp/codeowners_benchmarks.md
+fi
+
+echo "To run these benchmarks on your application, you can place this repo next to your rails application and run bash ../rubyatscale/codeowners-rs/dev/run_benchmarks.sh from the root of your application" >> tmp/codeowners_benchmarks.md
```

Comment:
I had to make this script executable to follow these steps:
```sh
chmod +x ../rubyatscale/codeowners-rs/dev/run_benchmarks.sh
```

Consider updating instructions, or adding in the shell, eg:
```sh
/usr/bin/env bash ../rubyatscale/codeowners-rs/dev/run_benchmarks.sh
```